Project.toml

```diff
@@ -13,6 +13,7 @@ ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
 ImageTransformations = "02fcd773-0e25-5acc-982a-7f6622650795"
 Interpolations = "a98d9a8b-a2ab-59e6-89dd-64a1c18fca59"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Rotations = "6038ab10-8711-5258-84ad-4b1120ba62dc"
```

Comment:
Again, every dependency should declare its compatibility.

Actually, there's a way to work to make all these indirect dependencies. Say we don't add `Rotations` to Project.toml, we can still get everything via:

```julia
using ImageTransformations.Rotations
```

Reply:
Yeah I have updated all the compatibilities in the main repo, this repo was branched from the trfm branch just before I had made those changes. I have added all the compats.